extensions/amp-ad-network-fake-impl/0.1/data/fake_amp_ad_with_3p_analytics.json

```diff
@@ -0,0 +1 @@
+<!-- No, this is not actually JSON. Fake ad network is kind of broken and should be fixed separately. --><!doctype html><html amp4ads> <head> <meta charset=utf-8> <script async src='https://cdn.ampproject.org/v0.js'></script> <script async custom-element=amp-analytics src='https://cdn.ampproject.org/v0/amp-analytics-0.1.js'></script> <style amp4ads-boilerplate>body{visibility:hidden}</style> <meta name=viewport content=width=device-width,minimum-scale=1> </head> <body> <amp-img id=img src='https://upload.wikimedia.org/wikipedia/commons/6/6e/Golde33443.jpg' width=256 height=300></amp-img> <amp-analytics> <script type='application/json'> { "requestsFrameUrl": "http://localhost:8000/examples/analytics-3p-remote-frame.html", "dataHash": "key1=val1&key2=val2", "requests": { "pageview": "https://example.com/analytics?viewed=true&url=${canonicalUrl}&title=${title}&acct=${account}", "event": "https://example.com/analytics?eid=click&acct=${account}" }, "vars": { "account": "ABC123" }, "triggers": { "defaultPageview": { "on": "visible", "request": "pageview" }, "anchorClicks": { "on": "click", "selector": "img", "request": "event" } } } </script> </amp-analytics> </body></html>
```

Comment:
See comment above beginning with "Fake ad network is broken!". I'll go fix it in a separate PR, and then correct this here. But tl;dr if you put JSON like it says you're supposed to, you see the raw JSON in your ad, and if you try to specify <amp-ad type="fake" src="foo.html"> rather than foo.json, then you see nothing at all.